activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -538,6 +554,10 @@ def default_index_type?(index) # :nodoc:
         index.using.nil?
       end
 
+      def insert_all_sql_builder
+        ActiveRecord::InsertAll::SqlBuilders::Generic
+      end
```

Comment:
I think I'd prefer just to do a const lookup in InsertAll. Lookup on convention of where there's a ActiveRecord::InsertAll::SqlBuilders::<adapter-name> builder available, if so, use that. Otherwise use the generic.

Reply:
Yeah, fine now.